configure.ac

```diff
@@ -37,6 +37,14 @@ LIBYAMI_LT_LDFLAGS="-version-number $LIBYAMI_LT_VERSION"
 AC_SUBST(LIBYAMI_LT_VERSION)
 AC_SUBST(LIBYAMI_LT_LDFLAGS)
 
+# check for required macro definitions
+AC_MSG_CHECKING([for ax_pkg_check_modules])
+m4_ifndef([AX_PKG_CHECK_MODULES],
+    [AC_MSG_RESULT([no])
+     m4_defun([AX_PKG_CHECK_MODULES],[])
+     AC_MSG_ERROR([autoconf-archive >= 2015.02.24 is required])],
```

Comment:
This may introduce inconvenience for user. Could we rewrite this function? or other choose?